packages/common/src/types/filter.ts

```diff
@@ -66,8 +66,19 @@ export type DashboardFilterRule<
 > = FilterRule<O, T, V, S> & {
     tileTargets?: Record<string, DashboardTileTarget>;
     label: undefined | string;
+    required?: boolean;
 };
 
+/**
+ * DashboardFilterRuleForTile is a DashboardFilterRule without properties that are unknown to FilterRule: e.g. `tileTargets`, `label`, and `required` fields
+ * This is used to represent the filter rules that are applied to a specific tile.
+ * This is also to ensure we don't polute the filters on a chart tile with unnecessary fields that can't be used when viewing a chart in isolation.
+ */
+export type DashboardFilterRuleForTile = Pick<
```

Comment:
Isn't this type identical to `FilterRule` ?

- `MetricQuery` has `filters` which is type `Filters` contains `FilterRule`s
- `Dasbboard` has `dashboardFilters` has type `DashboardFilters` contains `DashboardFilterRule`s

The problem with the last code was that explore from here, was including dashboard filter rule properties as regular filter rules. We need to be explicit when we're casting from `DashboardFilterRule` to a simple `FilterRule`

Reply:
Almost identical! 
DashboardFilterRule has DashboardFieldTarget: 
```
export type DashboardFieldTarget = {
    fieldId: string;
    tableName: string;
};

```

whereas FilterRule has FieldTarget: 

```
export type FieldTarget = {
    fieldId: string;
};
```

Checking if making the complete conversion to FilterRule works, thanks!